docs/01-overview/main-areas/observability/obsv-03-tracing-in-kyma.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Tracing in Kyma
+---
+
+With the [Jaeger](https://github.com/jaegertracing) distributed tracing system, you can analyze the path of a request chain going through your distributed applications. This information helps you to, for example, troubleshoot your applications, or optimize the latency and performance of your solution.
+
```

Comment:
We wanted to add limits here as well: it has no persistence enabled and keeps all data in-memory with a retention of 10000 traces